src/System.Private.Xml.Linq/src/System.Private.Xml.Linq.csproj

```diff
@@ -9,15 +9,15 @@
     <DefineConstants>$(DefineConstants);SILVERLIGHT</DefineConstants>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'net46'">true</IsPartialFacadeAssembly>
     <ResourcesSourceOutputDirectory Condition="'$(TargetGroup)' == 'net46'">None</ResourcesSourceOutputDirectory>
```

Comment:
one-to-one in this context is that the implementation should be the same identity as the contract. Both hopefully can be System.Xml.XDocument. That can only happen if they have the same public surface area though.